js/ui/date_range_box.d.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  EventInfo,
+  NativeEventInfo,
+  InitializedEventInfo,
+  ChangedOptionInfo,
+} from '../events/index';
+
+import {
+  DropDownButtonTemplateDataModel,
+} from './drop_down_editor/ui.drop_down_editor';
+
+import dxDateBox, { dxDateBoxOptions } from './date_box';
+
+import {
+  ValueChangedInfo,
+} from './editor/editor';
+
+/** @public */
+export type DateRangePickerType = 'calendar' | 'native';
+
+/** @public */
+export type ChangeEvent = NativeEventInfo<dxDateRangeBox>;
+
+/** @public */
+export type ClosedEvent = EventInfo<dxDateRangeBox>;
+
+/** @public */
+export type ContentReadyEvent = EventInfo<dxDateRangeBox>;
+
+/** @public */
+export type CopyEvent = NativeEventInfo<dxDateRangeBox, ClipboardEvent>;
+
+/** @public */
+export type CutEvent = NativeEventInfo<dxDateRangeBox, ClipboardEvent>;
+
+/** @public */
+export type DisposingEvent = EventInfo<dxDateRangeBox>;
+
+/** @public */
+export type EnterKeyEvent = NativeEventInfo<dxDateRangeBox, KeyboardEvent>;
+
+/** @public */
+export type FocusInEvent = NativeEventInfo<dxDateRangeBox, FocusEvent>;
+
+/** @public */
+export type FocusOutEvent = NativeEventInfo<dxDateRangeBox, FocusEvent>;
+
+/** @public */
+export type InitializedEvent = InitializedEventInfo<dxDateRangeBox>;
+
+/** @public */
+export type InputEvent = NativeEventInfo<dxDateRangeBox, UIEvent & { target: HTMLInputElement }>;
+
+/** @public */
+export type KeyDownEvent = NativeEventInfo<dxDateRangeBox, KeyboardEvent>;
+
+/** @public */
+export type KeyPressEvent = NativeEventInfo<dxDateRangeBox, KeyboardEvent>;
+
+/** @public */
+export type KeyUpEvent = NativeEventInfo<dxDateRangeBox, KeyboardEvent>;
+
+/** @public */
+export type OpenedEvent = EventInfo<dxDateRangeBox>;
+
+/** @public */
+export type OptionChangedEvent = EventInfo<dxDateRangeBox> & ChangedOptionInfo;
+
+/** @public */
+export type PasteEvent = NativeEventInfo<dxDateRangeBox, ClipboardEvent>;
+
+/** @public */
+export type ValueChangedEvent =
+    NativeEventInfo<dxDateRangeBox, KeyboardEvent | MouseEvent | PointerEvent | Event>
+    & ValueChangedInfo;
+
+/** @public */
+export type DropDownButtonTemplateData = DropDownButtonTemplateDataModel;
+
+/**
+ * @deprecated use Properties instead
+ * @namespace DevExpress.ui
+ */
+export interface dxDateRangeBoxOptions extends Omit<dxDateBoxOptions<dxDateRangeBox>, 'value'> {
+  /**
+     * @docid
```

Comment:
Specify docid values for all nested members after removing dxDateRangeBoxOptions with this prefix:
@docid dxDateRangeBoxOptions.endDate
etc.